.github/workflows/release.yml

```diff
@@ -0,0 +1,214 @@
+    tags:
+      - "v*" # Run when tag matches v*, i.e. v1.0, v20.15.10
+
+name: Release
+
+env:
+  RELEASE_BIN: wrangler
+  RELEASE_DIR: artifacts
+  GITHUB_REF: "${{ github.ref }}"
+  WINDOWS_TARGET: x86_64-pc-windows-msvc
+  MACOS_TARGET: x86_64-apple-darwin
+  LINUX_TARGET: x86_64-unknown-linux-musl
+
+  # Space separated paths to include in the archive.
+  RELEASE_ADDS: README.md LICENSE-APACHE LICENSE-MIT wrangler-demo.gif banner.png
+
+jobs:
+  build:
+    name: Build artifacts
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        build: [linux, macos, windows, wranglerjs]
+        include:
+          - build: linux
+            os: ubuntu-latest
+            rust: stable
+          - build: macos
+            os: macos-latest
+            rust: stable
+          - build: windows
+            os: windows-latest
+            rust: stable
+          - build: wranglerjs
+            os: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Cache Cargo registry
+        uses: actions/cache@v1
+        if: matrix.rust
+        with:
+          path: ~/.cargo/registry
+          key: ${{ matrix.build }}-stable-cargo-registry-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: |
+            ${{ matrix.build }}-stable-cargo-registry-
+
+      - name: Cache Cargo index
+        uses: actions/cache@v1
+        if: matrix.rust
+        with:
+          path: ~/.cargo/git
+          key: ${{ matrix.build }}-stable-cargo-index-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: |
+            ${{ matrix.build }}-stable-cargo-index-
+
+      - name: Cache Cargo build
+        uses: actions/cache@v1
+        if: matrix.rust
+        with:
+          path: target/release
+          key: ${{ matrix.build }}-stable-release-target-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: |
+            ${{ matrix.build }}-stable-release-target-
+
+      - name: Query version number
+        id: get_version
+        shell: bash
+        run: |
+          echo "using version tag ${GITHUB_REF:10}"
+          echo ::set-output name=version::"${GITHUB_REF:10}"
+
+      - name: Install Rust
+        if: matrix.rust
+        run: |
+          rustup update ${{ matrix.rust }} --no-self-update
+          rustup default ${{ matrix.rust }}
+
+      - name: Install musl-tools (Linux)
+        if: matrix.build == 'linux'
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install musl-tools -y
+
+      - name: Install p7zip (MacOS)
+        if: matrix.build == 'macos'
+        run: brew install p7zip
+
+      - name: Build (Linux)
+        if: matrix.build == 'linux'
+        run: |
+          rustup target add ${{ env.LINUX_TARGET }}
+          cargo build --release --target ${{ env.LINUX_TARGET }} --features vendored-openssl
+
+      - name: Build (MacOS)
+        if: matrix.build == 'macos'
+        run: cargo build --release
+        env:
+          MACOSX_DEPLOYMENT_TARGET: 10.7
```

Comment:
how'd we pick this number? was it what we used previously?

Reply:
this is what was in azure and why i asked if we needed it

Reply:
got it. i don't think we need it! apple is particularly shitty about what counts as latest stable... based off of what still gets security updates (https://support.apple.com/en-us/HT201222), i think we should aim for 10.14 right now, but this feels like a guess on my part, so i'm open to thoughts you have!